main.py

```diff
@@ -3,14 +3,27 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    while True:
```

Comment:
**Suggestion:** Limit the frame rate of the game loop to improve performance.

**Existing Code:**
```Python
while True:
```

**Improved Code:**
```Python
clock = pygame.time.Clock()
while True:
    clock.tick(60)  # Limit to 60 frames per second
```

**Details:**
Implement a mechanism to limit the frame rate of the game loop to improve performance and reduce CPU usage.